p2p_wallet/Models/PendingTransaction.swift

```diff
@@ -6,76 +6,10 @@
 //
 
 import Foundation
-import RxSwift
```

Comment:
I saw you moved it to submodule but have pods already been working or it is not in the branch?